src/Components/Publishing/Sections/Text.tsx

```diff
@@ -25,6 +25,7 @@ export class Text extends Component<Props, State> {
   static defaultProps = {
     color: "black",
     showTooltips: false,
+    isMobile: false,
```

Comment:
I think this is unused now and we are just using showTooltips. Should be able to remove, I think I missed it 🙈

Reply:
Done with d263255.